contrib/proguard/src/Proguard.scala

```diff
@@ -60,26 +92,38 @@ trait Proguard extends ScalaModule {
     outJar
```

Comment:
Following the change above...

```suggestion
    PathRef(outJar)
```